spec/services/idv/agent_spec.rb

```diff
@@ -151,6 +158,33 @@
           timed_out: true,
         )
       end
+
+      context 'successfully proofs in IPP flow' do
+        let(:ipp_enrollment_in_progress) { true }
+
+        it 'returns a successful result' do
+          # rubocop:disable Layout/LineLength
+          agent = Idv::Agent.new(Idp::Constants::MOCK_IDV_APPLICANT_STATE_ID_ADDRESS.merge(uuid: user.uuid))
+          # rubocop:enable Layout/LineLength
```

Comment:
No this is a good call out. I use `rubocop` as a crutch sometimes and should reformat/refactor instead of just disabling the linter, since it's just doing it's job.